src/train.py

```
@@ -0,0 +1,189 @@
+    save_history_plot(history, os.path.join(RUN_RESULTS_DIR, "history.png"))
+    save_misclassified_plot(model, X_val, y_val_labels,
+                            os.path.join(RUN_RESULTS_DIR, "misclassified.png"))
+
+    # Riport (validációs adatokon)
+    report_path = os.path.join(RUN_RESULTS_DIR, "validation_report.txt")
+    try:
+        val_pred_probs = model.predict(X_val)
+        val_pred = np.argmax(val_pred_probs, axis=1)
+        report = classification_report(y_val_labels, val_pred)
+        with open(report_path, 'w') as f:
+            f.write(f"--- Model information ---\n")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)